### PR TITLE
string methods

### LINKED ISSUE #4136: String Methods in Javascript
### Description

Useful Methods that can be used in String objects in Javascript while front-end development.

Methods:
1. length
2. slice()
3. substring()
4. replace()
5. toUpperCase()
6. toLowerCase()
7. concat()
8. trim()
9. charAt()
10. charCodeAt()
11. split()

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by itsajay1029**
<hr>

## Description 📜

Useful Methods that can be used in String objects in Javascript while front-end development.

Methods:

1.length
2.slice()
3.substring()
4.replace()
5.toUpperCase()
6.toLowerCase()
7.concat()
8.trim()
9.charAt()
10.charCodeAt()
11.split()
Fixes #4136 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add output also for every method.

File: Web_Development/Javascript/String Methods/String_method.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+
+
```
[the remark this answers is not in the document]
Comment (itsajay1029) [reply Web_Development/Javascript/String Methods/String_method.md]: Done.

**Review by kashishahuja2002 — APPROVED** [review]
Simple yet sorted work, good. 😃

**Comment by kashishahuja2002** [comment]
> Good to go. @praveenscience

Please further review and merge the PR. @praveenscience

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**